nbs/example.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>Small nit need to update these to <code>#|</code> which is the new style instead of <code>#</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/811/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>